src/tactic/ui/table/statistic_wdg.py

```diff
@@ -518,13 +518,24 @@ def init_data(my):
 
         due_date = parser.parse(value)
 
+        import calendar
+        from datetime import datetime, timedelta
+
+        # convert the due date from UTC to local time
+        timestamp = calendar.timegm(due_date.timetuple())
+        local_dt = datetime.fromtimestamp(timestamp)
+        assert due_date.resolution >= timedelta(microseconds=1)
+        due_date = local_dt.replace(microsecond=due_date.microsecond)
+
         # get today's date
         from pyasm.common import SPTDate
-        today = SPTDate.start_of_today()
+        import datetime
+        import time
+        today = datetime.datetime.now()
 
         # get the difference
         delta = due_date - today
-        diff = delta.days
+        diff = (delta.days * 86400) + delta.seconds
```

Comment:
should define a class constant called DAY = 86400, HOUR = 3600... so you can do DAY * 2 later or HOUR *2